Phase #3/app/Constants.cs

```diff
@@ -0,0 +1,9 @@
+
+public class Constants
+{
+     public static string STUDENTS_FILE_ADDRESS = @"resources/students.json";
```

Comment:
https://stackoverflow.com/questions/44920987/normal-naming-convention-for-a-public-static-variable-in-a-class